views/includes/confirmation-statement-details/sic-details.html

```diff
@@ -0,0 +1,7 @@
+<p class="govuk-body-m">
```

Comment:
for the folder naming, i think views/includes/sic/sic-details.html might be better, then you could group all the sic includes in that folder.